JBKLabs/react-models
# 2 infinite loop in hook around usecallback

Closes #2 

## Changes

* fix local dev issue where `package.json` was excluded
* patch infinite render loop caused by internal transform callback

## Steps to Test

In one terminal at root:
* `npm i`
* `npm run dev`

In a second terminal in `example`:
* `npm i`
* [x] `npm start` --> should work with no module errors

Additional steps:
* make the following changes to `buildModel.js`
![image](https://user-images.githubusercontent.com/12001504/71738897-9bbcea80-2e1d-11ea-93de-835f8bbb516b.png)
* [x] open up the example app console in your browser --> infinite log loop
* make the following changes to `buildModel.js`
![image](https://user-images.githubusercontent.com/12001504/71738920-b0997e00-2e1d-11ea-9231-451b3070b6fd.png)
* [x] open up the example app console in your browser --> infinite log loop should be gone
